ush/generate_FV3LAM_wflow.sh

```diff
@@ -360,55 +376,54 @@ done
 #
 #-----------------------------------------------------------------------
 #
-# For select workflow tasks, copy the system-specific environment and
```

Comment:
I  think there may be some issues with the version of your fork compared to mine (yours may not be updated)? I can update tomorrow and then have you update once more and then we can merge. Does that sound good?